manifest_staging/charts/secrets-store-csi-driver/templates/_helpers.tpl

```diff
@@ -12,13 +12,17 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 If release name contains chart name it will be used as a full name.
 */}}
 {{- define "sscd.fullname" -}}
+{{- if .Values.fullnameOverride -}}
```

Comment:
Can we add this in doc for default value?

Reply:
Did you mean adding the value to the helm configuration table?

`nameOverride` and `fullnameOverride` are already in the [configuration table](https://github.com/kubernetes-sigs/secrets-store-csi-driver/tree/master/manifest_staging/charts/secrets-store-csi-driver#configuration). The default value is empty because we don't explicitly set anything in the `values.yaml`.